app/code/Magento/CatalogInventory/Api/StockQtyDecimalInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
```

Comment:
Please enable the strict type mode with `declare(strict_types=1);` and specify strict types for parameters